run_tests

```diff
@@ -1,22 +1,19 @@
-#!/bin/bash
-# Copyright 2020 dylan
+#!/bin/sh -e
```

Comment:
Whatever you prefer here but I would maybe consider `set -e` rather than changing the shell

Reply:
I think I used [this](https://github.com/jnsgruk/charm-kubernetes-dashboard/blob/master/run_tests) as a reference at the time. `/bin/sh` might be more portable if `bash` functionality is not need. Either way, the preferred shebang for bash seems to be `#!/usr/bin/env bash`.